src/mem/cache/prefetch/indirect_memory.cc

```diff
@@ -44,11 +44,15 @@ IndirectMemory::IndirectMemory(const IndirectMemoryPrefetcherParams &p)
     shiftValues(p.shift_values), prefetchThreshold(p.prefetch_threshold),
     streamCounterThreshold(p.stream_counter_threshold),
     streamingDistance(p.streaming_distance),
-    prefetchTable(p.pt_table_assoc, p.pt_table_entries,
-                  p.pt_table_indexing_policy, p.pt_table_replacement_policy,
+    prefetchTable("PrefetchTable",
+		  p.pt_table_entries,
+		  p.pt_table_assoc,
+		  p.pt_table_replacement_policy,
+                  p.pt_table_indexing_policy,
                   PrefetchTableEntry(p.num_indirect_counter_bits)),
-    ipd(p.ipd_table_assoc, p.ipd_table_entries, p.ipd_table_indexing_policy,
+    ipd("IPD", p.ipd_table_entries, p.ipd_table_assoc,
```

Comment:
Why do we want the caches to be named? Also, as it is the names are not unique. Shouldn´t we use the name of the SimObject instantiating the cache to form a unique name?

Reply:
Bump

Reply:
I am adding the name of the SimObject to the objects name so that they are not unique.